src/main/java/com/palantir/gradle/versions/VersionsPropsPlugin.java

```diff
@@ -92,6 +92,25 @@ public final void apply(Project project) {
                     });
             project.getTasks().named("check").configure(task -> task.dependsOn(checkNoUnusedConstraints));
 
+            String overbroadConstraintsFlag = project.findProperty("gcv.overbroad.constraints") != null
+                    ? project.findProperty("gcv.overbroad.constraints").toString()
+                    : "false";
+
+            if (Boolean.parseBoolean(overbroadConstraintsFlag)) {
```

Comment:
Actually, new plan. We don't need the feature flag. To test in the background we just need to make the task but not do the `check.dependsOn` bit. This means the task will be there but won't be run unless we explicitly call it.